test/merge-patient-lab-iips-mocha.js

```diff
@@ -8,181 +9,146 @@ chai.use(chaiAsPromised);
 var _ = require('underscore');
 var noflo = require('noflo');
 
-var componentFactory = require('../components/merge-patient-lab-iips');
-var getComponent = componentFactory.getComponent;
+var compFactory = require('../components/merge-patient-lab-iips');
 
-var commonTest = require('./common-test');
+var test = require('./common-test');
 var commonStubs = require('./common-stubs');
 
 describe('merge-patient-lab-iips', function() {
-
-    describe('#rpf', function() {
  
-        it("should have an rpf object on nodeInstance", function() {
-            return Promise.resolve(componentFactory.getComponent )
-                .then(commonTest.createComponent).then(function(component){
-                    component.rpf.should.exist;
-                    component.rpf.should.be.an('object');
-            });
-        });
+    it("should be an object on nodeInstance", function() {
+        var node = test.createComponent(compFactory);
```

Comment:
yes, but since it is only one line of code, I'd rather keep it as a test-local variable.